core/src/main/java/org/elasticsearch/common/geo/GeoUtils.java

```diff
@@ -454,6 +439,27 @@ public static GeoPoint parseGeoPoint(XContentParser parser, GeoPoint point) thro
         }
     }
 
+    public static GeoPoint parseNumericArray(XContentParser parser, GeoPoint point) throws IOException {
+        double lat = Double.NaN;
+        double lon = Double.NaN;
+        int element = 0;
+        while(parser.nextToken() != Token.END_ARRAY) {
+            if(parser.currentToken() == Token.VALUE_NUMBER) {
+                element++;
+                if(element == 1) {
+                    lon = parser.doubleValue();
```

Comment:
Hmm do we also do `lon` then `lat` elsewhere in the ES APIs?  This is very confusing ... Lucene standardized on `lat` then `lon` a while back!